xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/mockData.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+define({
+    "totalrows": 5,
+    "returnedrows": 5,
+    "offset": 1,
+    "sort": [{property: "age", dir: "desc"}],
+    layouts: ["table"],
+    currentLayout: "table",
+
+    properties: [
```

Comment:
I don't think you can mix ``properties`` with ``propertyDescriptors``. We need both, as they serve different roles. If you're not convinced we can discuss.

Reply:
Yes I totally agree, was just waiting the updated specs, now I have them I'll use them from now :)